### PR TITLE
change image for bolognese

### PR DESCRIPTION
**Pull request by plpnk23**

### CODE REVIEW

File: recipes/spaghetti-bolognese.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
     <h1>Spaghetti Bolognese</h1>
-    <img src="images/spaghettibolognese.jpeg">
+    <img height="300" src="images/Spagetti-Putaneska.jpeg">
```
Comment (DmytroRiabov) [inline recipes/spaghetti-bolognese.html]: file names should be kebab-case(e.g. spagetti-putaneska)

File: recipes/spaghetti-bolognese.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
 </head>
 <body>
     <h1>Spaghetti Bolognese</h1>
-    <img src="images/spaghettibolognese.jpeg">
```
Comment (DmytroRiabov) [inline recipes/spaghetti-bolognese.html:11]: make sure you dont have any unneeded files in images folder

**Review by DmytroRiabov — CHANGES_REQUESTED** [review]

File: recipes/spaghetti-bolognese.html (hunk 1 above)
Comment (plpnk23) [reply recipes/spaghetti-bolognese.html]: addressed

File: recipes/spaghetti-bolognese.html (hunk 2 above)
Comment (plpnk23) [reply recipes/spaghetti-bolognese.html:11]: addressed

**Review by DmytroRiabov — APPROVED** [review]

**Pull request merged**